pgcli/main.py

```diff
@@ -642,7 +644,15 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
     database = database or dbname
     user = username or user
 
-    if '://' in database:
+    if dsn is not '':
+        try:
+            cfg = load_config(config_full_path)
+            dsn_config = cfg['dsns'][dsn]
+            pgcli.connect_uri(dsn_config)
+        except:
+            print ('DSN sections is not valid into config file')
```

Comment:
Can you use [`click.secho()`](http://click.pocoo.org/5/api/#click.secho) to print this error message instead of using `print()`? 

Also can you change the error message to be more descriptive as follows:

Invalid DSNs found in the config file. Please check the "[alias_dsn]" section in pgclirc.